plugin/src/test/java/com/h3xstream/findsecbugs/kotlin/KotlinHardcodedPasswordEqualsDetectorTest.java

```diff
@@ -0,0 +1,44 @@
+package com.h3xstream.findsecbugs.kotlin;
```

Comment:
At the moment the build is failing because the license header is missing.

```
[INFO] Checking licenses...
[WARNING] Missing header in: /home/travis/build/find-sec-bugs/find-sec-bugs/plugin-deps/src/main/java/org/jetbrains/annotations/NotNull.java
[WARNING] Missing header in: /home/travis/build/find-sec-bugs/find-sec-bugs/plugin-deps/src/main/java/kotlin/Metadata.java
[WARNING] Missing header in: /home/travis/build/find-sec-bugs/find-sec-bugs/plugin-deps/src/main/java/kotlin/jvm/internal/Intrinsics.java
[WARNING] Missing header in: /home/travis/build/find-sec-bugs/find-sec-bugs/plugin/src/main/java/com/h3xstream/findsecbugs/password/AbstractHardcodedPasswordEqualsDetector.java
[WARNING] Missing header in: /home/travis/build/find-sec-bugs/find-sec-bugs/plugin/src/main/java/com/h3xstream/findsecbugs/kotlin/KotlinHardcodedPasswordEqualsDetector.java
[WARNING] Missing header in: /home/travis/build/find-sec-bugs/find-sec-bugs/plugin/src/test/java/com/h3xstream/findsecbugs/kotlin/KotlinHardcodedPasswordEqualsDetectorTest.java
```

 - This check is made for Test and Detector classes. You can copy paster the license header from any other class.

 - All stub class in plugin-deps can be whitelist to avoid this check --> https://github.com/find-sec-bugs/find-sec-bugs/blob/master/pom.xml#L56-L82. (You can add pattern to cover future classes `/java/kotlin/**`, `org/jetbrains/**`)

Reply:
Sorry about that. I just ran the individual tests locally. Fixed the license headers now.